backend/pkg/traces/server.go

```diff
@@ -55,10 +65,19 @@ func CreateHTTPTracesServer(port int, traceHandleFunc HandleTraceFunc) (*HTTPTra
 
 	server.ConfigureFlags()
 	server.ConfigureAPI()
+	server.Port = config.Port
+
+	// We want to serve both http and https
+	// TODO: need to use istio to secure the http port when the wasm is sending traces
```

Comment:
maybe this todo should be in the wasm filter config? how does this influence the traces server?

Reply:
need to inject envoy sidecar on apiclarity deployment (the traces server)